helm/pattern-1/README.md

```diff
@@ -0,0 +1,101 @@
+# Helm Charts for deployment of WSO2 API Manager with Analytics
+
+## Prerequisites
+
+* In order to use WSO2 Helm resources, you need an active WSO2 subscription. If you do not possess an active WSO2
+  subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription).<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). This can
+ be easily done via 
+  ```
+  helm install stable/nginx-ingress --name nginx-wso2apim-analytics --set rbac.create=true
+  ```
+## Quick Start Guide
+>In the context of this document, <br>
+>* `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-apim`](https://github.com/wso2/kubernetes-apim/)
+Git repository. <br>
+>* `HELM_HOME` will refer to `<KUBERNETES_HOME>/helm/pattern-1`. <br>
+
+##### 1. Checkout Kubernetes Resources for WSO2 API Manager Git repository:
+
+```
+git clone https://github.com/wso2/kubernetes-apim.git
+```
+
+##### 2. Provide configurations:
+
+1. The default product configurations are available at `<HELM_HOME>/apim-with-analytics-conf/confs` folder. Change the 
+configurations as necessary.
+
+2. Open the `<HELM_HOME>/apim-with-analytics-conf/values.yaml` and provide the following values.
+
+    `username`: Username of your WSO2 subscription credentials<br>
+    `password`: Password of your WSO2 subscription credentials<br>
+    `email`: Docker email<br>
+    `namespace`: Namespace<br>
+    `svcaccount`: Service Account<br>
+    `serverIp`: NFS Server IP<br>
+    `locationPath`: NFS location path<br>
+    `sharedDeploymentLocationPath`: NFS shared deployment directory(<APIM_HOME>/repository/deployment) location for APIM<br> 
+    `analyticsDataLocationPath`: NFS volume for Indexed data for Analytics(<DAS_HOME>/repository/data)<br>
+    `analyticsLocationPath`: NFS volume for Analytics data for Analytics(<DAS_HOME>/repository/analytics)<br>
+    
+3. Open the `<HELM_HOME>/apim-with-analytics-deployment/values.yaml` and provide the following values.
+
+    `namespace`: Namespace<br>
+    `svcaccount`: Service Account
+    
+##### 3. Deploy the configurations:
+
+```
+helm install --name <RELEASE_NAME> <HELM_HOME>/apim-with-analytics-conf
+```
+
+##### 4. Deploy MySql:
```

Comment:
@ThilinaManamgoda Shouldn't it be MySQL?

Reply:
Yes, it should. I will fix it.